libview/ev-view.c

```diff
@@ -312,7 +326,8 @@ static void       ev_view_primary_clear_cb                   (GtkClipboard
 static void       ev_view_update_primary_selection           (EvView             *ev_view);
 
 #if GTK_CHECK_VERSION (3, 0, 0)
-G_DEFINE_TYPE_WITH_CODE (EvView, ev_view, GTK_TYPE_FIXED, G_IMPLEMENT_INTERFACE (GTK_TYPE_SCROLLABLE, NULL))
+G_DEFINE_TYPE_WITH_CODE (EvView, ev_view, GTK_TYPE_CONTAINER,
+                         G_IMPLEMENT_INTERFACE (GTK_TYPE_SCROLLABLE, NULL))
```

Comment:
I wonder if we can make EvView derive from GtkContainer in GTK+2 build as well. This would make all this code a bit simpler and easier to read.